examples/power_spectra_example.py

```diff
@@ -0,0 +1,44 @@
+"""
+===============================
+Power Spectrum of a TimeSeries
+===============================
+
+An example showing how to estimate the power spectrum of a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import matplotlib.pyplot as plt
+import pandas as pd 
+from scipy import signal
+import astropy.units as u
+
+import sunpy.timeseries
+from sunpy.data.sample import RHESSI_TIMESERIES
+
+###############################################################################
+# Let's first load a TimeSeries from sample data.
+# Input data contains 9 columns, which are evenly sampled with a time
+# step of 4 seconds. So, the data is suitable for signal analysis.
+ts = sunpy.timeseries.TimeSeries(RHESSI_TIMESERIES)
+
+###############################################################################
+# We now use SciPy's periodogram to estimate the power spectra of the first 
+# column of the Timeseries. The first column contains X-Ray emmisions in the range 
+# of 3-6 keV. Astropy's Lomb-Scargle Periodograms 
+# (http://docs.astropy.org/en/v2.0.5/stats/lombscargle.html) can be used
```

Comment:
Try `~astropy.stats.LombScargle` instead. Sphinx should link to it since we have that setup. 

I would rephrase the last sentence as well. `An alterantive version is ....` or something.